intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -403,7 +441,7 @@ model_token Model::post_process(float* logits) {
 
 int Model::quant_model(const std::string& model_path, const std::string& out_path, const std::string& weight_dtype,
                        const std::string& alg, int group_size, const std::string& scale_dtype,
-                       const std::string& compute_dtype, bool use_ggml) {
+                       const std::string& compute_dtype, bool use_ggml, int threads) {
```

Comment:
for clients?

Reply:
There is a `int threads` in `generate` and `generate_tokens` and I guess they are no more "for clients" than `quant_model`.